server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -89,22 +90,30 @@ async function updateExistingSession(
         ) {
             // Previous session was in a different location. Move to current location.
             // Reset logOffset, ordererUrl, and historianUrl when moving session location.
-            Lumberjack.info(
-                `Reset logOffset, ordererUrl, and historianUrl when switching cluster.`,
-                lumberjackProperties,
-            );
+            Lumberjack.info("Moving session", {
+                ...lumberjackProperties,
+                isSessionSticky,
+                documentLastAccessTime: document.lastAccessTime,
+                sessionStickyCalculationTimestamp,
```

Comment:
Do you also want to pre-compute and add a property indicating how much is the current session sticky time?
Something like:
`currentSessionStickinessDurationMs: sessionStickyCalculationTimestamp - document.lastAccessTime`

Reply:
Yeah. This might need to handle the null cases. So we just put them into the properties.